seed/models/properties.py

```diff
@@ -231,6 +231,17 @@ class PropertyState(models.Model):
                                          null=True)
     analysis_state_message = models.TextField(null=True)
 
+    # stores results from various analyses in the following format:
+    # analyses_results = {
+    #     analysis_type: [ # ordered descending by timestamp
+    #         timestamp:
+    #         analysis_property_view_id:
+    #         data: { # depends on each analysis }
+    #         summary: # one-line summary for display
+    #     ]
+    # }
+    analyses_results = JSONField(default=dict, blank=True)
+
```

Comment:
Could we remove this and the migration for now? I do like this approach, but would like to hold off until we have the user discussion and time to chat about the pros and cons of different methods